src/diffusers/utils/dummy_torch_and_transformers_and_onnx_objects.py

```diff
@@ -4,6 +4,7 @@
 
 class OnnxStableDiffusionImg2ImgPipeline(metaclass=DummyObject):
     _backends = ["torch", "transformers", "onnx"]
+    _is_onnx = True
```

Comment:
I don't think we need this hard code this flag. When running these tests with `onnxruntime` installed will result in this flag being set to true.